blog/2023-06-16-inline-sdk.mdx

```diff
@@ -0,0 +1,1104 @@
+            </div>
+        </div>
+    </div>
+);
+```
+
+With everything added your component should look like this:
+
+```tsx title="src/bands/VideoBand/Band.tsx"
+import debounce from "lodash.debounce";
+import {
+    BiPlay,
+    BiPause,
+    BiVolumeFull,
+    BiVolumeMute,
+} from "react-icons/bi";
+import React, { useEffect, useRef } from "react";
+import useVideoPlayer from "./useVideoPlayer";
+import "./Band.css";
+import { BandProps } from "inline";
+
+export const Band: React.FC<BandProps> = (props) => {
+    const player = useRef<HTMLVideoElement>();
+    const {
+        playerState,
+        togglePlay,
+        handleOnTimeUpdate,
+        handleVideoSpeed,
+        toggleMute,
+        seekTo,
+    } = useVideoPlayer(player);
+
+    const handleVideoProgress = (
+        event: React.ChangeEvent<HTMLInputElement>
+    ) => {
+        const value = isNaN(+event.target.value)
+            ? 0
+            : +event.target.value;
+        seekTo(value);
+    };
+
+    return (
+        <div className="video-band container">
+            <div className="video-wrapper">
+                <video
+                    ref={player}
+                    onTimeUpdate={handleOnTimeUpdate}
+                    src={props.band.customBandSettings.url ?? ""}
+                />
+                <div className="controls">
+                    <div className="actions">
+                        <button onClick={togglePlay}>
+                            {!playerState.isPlaying ? (
+                                <BiPlay color="white" />
+                            ) : (
+                                <BiPause color="white" />
+                            )}
+                        </button>
+                    </div>
+                    <input
+                        type="range"
+                        min="0"
+                        max="100"
+                        value={playerState.progress}
+                        onChange={handleVideoProgress}
+                    />
+                    <select
+                        className="velocity"
+                        value={playerState.speed}
+                        onChange={handleVideoSpeed}
+                    >
+                        <option value="0.50">0.50</option>
+                        <option value="1">1</option>
+                        <option value="1.25">1.25</option>
+                        <option value="2">2</option>
+                    </select>
+                    <button className="mute-btn" onClick={toggleMute}>
+                        {!playerState.isMuted ? (
+                            <BiVolumeFull color="white" />
+                        ) : (
+                            <BiVolumeMute color="white" />
+                        )}
+                    </button>
+                </div>
+            </div>
+        </div>
+    );
+};
+```
+
+With the band created lets add some styling. Add this to the `src/bands/VideoBand/Band.css` file:
```

Comment:
It might be worth a "Styling the component" heading here.

Reply:
Fixed